src/components/ButtonGroup/ButtonGroup.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import ButtonGroup from './ButtonGroup';
+
+describe('Button Group component', () => {
+  beforeEach(() => {
+    render(
+      <ButtonGroup
+        buttons={['All Topics', 'Ukraine', 'COVID-19']}
+        value="All Topics"
+        onClick={() => console.log('buttonGroup')}
+        title="Discover recent fact checks on:"
+      />
+    );
+  });
+
+  it('should render all topics buttons', () => {
+    expect(screen.getByText('All Topics')).toBeInTheDocument();
+  });
+
+  it('should render ukraine button', () => {
+    expect(screen.getByText('Ukraine')).toBeInTheDocument();
+  });
+
+  it('should render covid-19 button', () => {
+    expect(screen.getByText('COVID-19')).toBeInTheDocument();
+  });
+
+  it('should render button group title', () => {
+    expect(
+      screen.getByText(/discover recent fact checks on/i)
+    ).toBeInTheDocument();
+  });
```

Comment:
Add a test for userEvent, similar to Button.test.tx:

```
const mockOnClick = jest.fn();
  it('should trigger onClick when clicked.', () => {
    render(<Button onClick={mockOnClick}>Test Button</Button>);
    // userEvent.click(screen.getByText(/Test Button/));''
    fireEvent.click(screen.getByText(/Test Button/));
    expect(mockOnClick).toHaveBeenCalledTimes(1);
  });
```

Reply:
But should we add testing to button that is already tested on Button component?

Reply:
We can negate this test for now. Agreed.